README.md

```diff
@@ -191,7 +191,7 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/litmus
 
 ## License
 
-© 2015 Litmus Software
+© 2021 Litmus Software
```

Comment:
@rahim we should do our future selves a solid and remove the timestamp as it's not a legal requirement. (If someone wants to determine recency of this project, they'll look at commits, issues, releases, etc.)